Makefile

```
@@ -359,20 +359,25 @@ config-controller-gen:
-	$(SILENT)$(eval GOMOCK_REFLECT_DIRS=`find . -type d -name 'gomock_reflect_*'`)
-	$(SILENT)test -z $(GOMOCK_REFLECT_DIRS) || { echo "Found leftover gomock directories. Please remove them and rerun make deps!"; echo $(GOMOCK_REFLECT_DIRS); exit 1; }
-	$(SILENT)go mod tidy
 ifdef CI
-	$(SILENT)git diff --exit-code -- go.mod go.sum || { echo "go.mod/go.sum files were updated after running 'go mod tidy', run this command on your local machine and commit the results." ; exit 1 ; }
-endif
+# In CI, go.mod/go.sum must be committed — not auto-updated.
+# go mod tidy is validated by check-generated-files (scripts/ci/jobs/check-generated.sh).
+deps:
+	@echo "deps: CI is set — skipping go mod tidy (validated by check-generated-files)"
+else
+deps: $(shell git ls-files '*/go.mod' 'go.mod')
```

checkmake (0.3.2)

[warning] Target body for "deps" exceeds allowed length of 5 lines (11).

(maxbodylength)